### ISSUE TITLE
Predstavitev v tkinterju ali čemer kol že

### ISSUE THREAD

**Issue by Petkomat** [TStepi/Igre#8]
Človek, ki je naredu avtomatskega igralca, ma na svoji strani tud slikice za karte, vsako v svoji datoteki.

http://tarok.bocosoft.com/

Kliknt je pol treba še na doljemanje.

**Comment by TStepi** [comment]
Pravkar je ta računalniški igralec napovedal trulo, js mam pa monda v roki... :+1: 
EDIT: tudi palčke ni imel

**Comment by Petkomat** [comment]
A je prepogumen al je mojstrsko odigral?

**Comment by TStepi** [comment]
Prepogumen, trulo kr za brezveze napoveduje. Tut pri beraču se ni izkazu. Sicer pa precej solidno. Je pa točkovanje pri tem taroku drugačno kot smo vajeni.

**Comment by Petkomat** [comment]
No, js sm zdej mal igral ... Napovedu trulo, čeprav ni mel pagata. Enkrat je šu igrat 1 in ni vzel pagata iz talona, napovedu kralje, čeprav sem mel jst 2 ... Enkrat je šu RAČ2 igrat, js sm vrgu enga platelca, RAČ1 palčko, RAČ2 pa je pol pobral. Pa palčka sploh ni bil edini tarok v roki RAČ1.  Ne vem, kaj nej si mislm.

Po ene 15 igrah je 114:98:99 zame, s tem, da sem enkrat po nesreči kliknu na monda in ga pol zgubu. Tud prepogumno sm šu beračit:)

**Comment by Petkomat** [comment]
Sm že 100 zadi, nekomu so se usrale dobre karte.

**Comment by Petkomat** [comment]
Haha, tip je napovedu trulo pa kralje pa mam monda pa kralja.
